internal/pkg/warewulfd/warewulfd.go

```diff
@@ -37,9 +41,22 @@ func RunServer() error {
 	http.HandleFunc("/overlay-system/", SystemOverlaySend)
 	http.HandleFunc("/overlay-runtime", RuntimeOverlaySend)
 
+	conf, err := warewulfconf.New()
+	if err != nil {
+		wwlog.Printf(wwlog.ERROR, "Could not get Warewulf configuration: %s\n", err)
+		os.Exit(1)
```

Comment:
Instead of the wwlog error and then exit, can you do this instead:

`return errors.Wrap(err, "could not get Warewulf configuration")`

Reply:
sure, didn't do that on purpose. I didn't want to mix old and new error handling.
Might take till Monday, though.

Reply:
Yeah, I'm on a push to update all error handling, every one helps!